cypress/integration/eventStatus.cy.js

```diff
@@ -49,13 +49,12 @@ describe('event status', () => {
         setUpTable(event[DIMENSION_ID_SCHEDULED_DATE])
 
         expectTableToMatchRows([
-            'Active',
-            'Scheduled',
```

Comment:
If there's no `scheduled` item in the db any longer then what's the point of this test?